services/utils/conditional-headers/src/main/java/org/eclipse/ditto/services/utils/headers/conditional/ConditionalHeadersValidator.java

```diff
@@ -95,13 +107,36 @@ public void checkConditionalHeaders(final Command command,
 
         checkIfMatch(command, currentETagValue);
         checkIfNoneMatch(command, currentETagValue);
+
     }
 
     private boolean skipPreconditionHeaderCheck(final Command command, @Nullable final EntityTag
             currentETagValue) {
         return currentETagValue == null &&
                 (Command.Category.DELETE.equals(command.getCategory()) ||
-                        Command.Category.QUERY.equals(command.getCategory()));
+                        Command.Category.QUERY.equals(command.getCategory())) || skipDueToRelation(command);
+    }
+
+    /**
+     * Skip precondition check if the selected fields contain exempted fields (e.g. {@code _policy} for things
+     * because the revision of a thing does not change if its policy is updated).
+     */
+    private boolean skipDueToRelation(final Command command) {
+        return command.toJson().getValue(SELECTED_FIELDS)
```

Comment:
Did you consider using `instanceof` here in order to avoid the `toJson()` and again the parsing of the field selector later.

E.g.: 
```
@Nullable final JsonFieldSelector selectedFields;
if (command instanceof RetrieveThing) {
  selectedFields = ((RetrieveThing) command).getSelectedFields().orElse(null);
} else if (command instanceof RetrieveThings) {
  selectedFields = ((RetrieveThings) command).getSelectedFields().orElse(null);
} else {
  selectedFields = null;
}
```

Reply:
The current solution probably has a worse performance.
And as e.g. one "RetrieveThings" command with page size 200 multiplies to 200 single "RetrieveThing" commands the impact on 
* creating the json
* getting a field
* creating a new field selector from it
could really make performance worse than an `instanceof` check.

Reply:
@dguggemos what do you think?

Reply:
that makes sense 👍